slack-api-client/src/main/java/com/slack/api/methods/request/slack_lists/SlackListsAccessDeleteRequest.java

```diff
@@ -0,0 +1,39 @@
+package com.slack.api.methods.request.slack_lists;
+
+import com.google.gson.annotations.SerializedName;
+import com.slack.api.methods.SlackApiRequest;
+import java.util.List;
+import lombok.Builder;
+import lombok.Data;
+
+/**
+ * https://docs.slack.dev/reference/methods/slackLists.access.delete
+ */
+@Data
+@Builder
+public class SlackListsAccessDeleteRequest implements SlackApiRequest {
+
+    /**
+     * Authentication token. Requires scope: `lists:write`
+     */
+    private String token;
+
+    /**
+     * Encoded ID of the List.
+     */
+    @SerializedName("list_id")
```

Comment:
nit: when the snake_case to camelCase (vice versa) is straight-forward like this one, the SerializedName annotation is not required.